span-normalizer/span-normalizer/src/test/java/org/hypertrace/core/spannormalizer/fieldgenerators/HttpFieldsGeneratorTest.java

```diff
@@ -729,4 +730,8 @@ public void testPopulateOtherFields() {
     Assertions.assertEquals(
         "some-query-str=v1", eventBuilder.getHttpBuilder().getRequestBuilder().getQueryString());
   }
+
+  public void testHttpFieldGenerationForOtelSpan() {
```

Comment:
Was this meant to have some code in it?

Reply:
Yeah, I am adding cases test cases for this.